views/patterns/01-molecules/components/carousel.blade.php

```diff
@@ -0,0 +1,26 @@
+<section class="hero-carousel">
```

Comment:
This part is already in the `v3` branch and shouldn't be  part of this pull request.